hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/exports/FileSignTool.java

```diff
@@ -99,6 +102,8 @@ public class FileSignTool {
     private static final String ALIAS_PROPERTY = "alias";
     private static final String PASSWORD_PROPERTY = "password";
     private static final String DIR_PROPERTY = "dir";
+    private static final String APP_VERSION = "appVersion";
+
     private static final Logger LOGGER = LogManager.getLogger(FileSignTool.class);
```

Comment:
would you like to migrate this to slf4j?

Reply:
Probably need a new PR for replacing all log4j2 with slf4j on hedera repo

Reply:
I was thinking that the we were going to just add it as we changed other stuff.  Nathan assigned it to all three of us.